src/LinuxTracing/UprobesUnwindingVisitorDwarfUnwindingTest.cpp

```diff
@@ -0,0 +1,1814 @@
+                                  TestFixture::kTargetAddress2),
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::function_name,
+                                  TestFixture::kFunctionName2),
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::offset_in_function, 0),
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::module_name,
+                                  TestFixture::kTargetName)),
+          ::testing::AllOf(
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::absolute_address,
+                                  TestFixture::kTargetAddress3),
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::function_name,
+                                  TestFixture::kFunctionName3),
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::offset_in_function, 0),
+              ::testing::Property(&orbit_grpc_protos::FullAddressInfo::module_name,
+                                  TestFixture::kTargetName))));
+
+  EXPECT_EQ(unwinding_errors, 0);
+  EXPECT_EQ(discarded_samples_in_uretprobes_counter, 0);
+}
+
+REGISTER_TYPED_TEST_SUITE_P(
+    UprobesUnwindingVisitorDwarfUnwindingTest,
+    VisitValidStackSampleWithoutUprobesSendsCompleteCallstackAndAddressInfos,
+    VisitEmptyStackSampleWithoutUprobesDoesNothing,
+    VisitInvalidStackSampleWithoutUprobesSendsUnwindingErrorAndAddressInfos,
+    VisitSingleFrameStackSampleInFunctionToStopAtSendsCompleteCallstackAndAddressInfos,
+    VisitSingleFrameStackSampleOutsideOfAnyFunctionToStopAtSendsUnwindingErrorAndAddressInfos,
+    VisitSingleFrameStackSampleWithoutUprobesSendsUnwindingErrorAndAddressInfos,
+    VisitStackSampleStoppedAtUprobesSendsPatchingFailedCallstack,
+    VisitStackSampleStoppedAtUserSpaceInstrumentationTrampolineSendsPatchingFailedCallstack,
+    VisitStackSampleUsesLatestUserSpaceCallstack,
+    VisitStackSampleUsesUserSpaceCallstackOnlyFromSameThread, VisitStackSampleUsesUserSpaceStack,
+    VisitStackSampleUsesUserStackMemoryFromAllStreamIds,
+    VisitStackSampleWithinUprobeSendsInUprobesCallstack,
+    VisitStackSampleWithinUserSpaceInstrumentationLibraryButNotTrampolineSendsCompleteCallstack,
+    VisitStackSampleWithinUserSpaceInstrumentationTrampolineAndLibrarySendsInUserSpaceInstrumentationCallstack,
+    VisitStackSampleWithinUserSpaceInstrumentationTrampolineSendsInUserSpaceInstrumentationCallstack,
+    VisitTwoValidStackSamplesSendsAddressInfosOnlyOnce,
+    VisitValidStackSampleWithNullptrMapInfosSendsCompleteCallstackAndAddressInfosWithoutModuleName);
+
+using TestTypes = ::testing::Types<
+    std::pair<StackSamplePerfEvent, orbit_grpc_protos::FullCallstackSample>,
+    std::pair<SchedWakeupWithStackPerfEvent, orbit_grpc_protos::ThreadStateSliceCallstack>,
+    std::pair<SchedSwitchWithStackPerfEvent, orbit_grpc_protos::ThreadStateSliceCallstack>>;
+INSTANTIATE_TYPED_TEST_SUITE_P(UprobesUnwindingVisitorSampleTestNew,
```

Comment:
I don't think the prefix should be `UprobesUnwindingVisitorSampleTestNew`

Reply:
Oh that was a leftover. Thanks!